testing/src/test/java/org/apidesign/javadoc/testing/VerifyJavadocTest.java

```diff
@@ -237,6 +238,27 @@ public void testSnippetWithSpecialCharacters() throws Exception {
         assertSnippet(text, "dollarSnippet2", "MONEY_SIGNS = <em>\"$ € £\"</em>");
     }
 
+    @Test
+    public void testPackageListElementListExists() {
+        String packageListFileName = "javadoc-bundle-options/package-list";
```

Comment:
This test checks that the `element-list` file is generated rather than testing whether it is read. E.g. this test covers the change in `pom.xml` that defines the `java-10` profile, but not the change in  `readPackageListFromFile` code, right?

To test the `readPackageListFromFile` change it might be enough to point the `links` section to official JDK-11 javadoc and check that the Javadoc can resolve some `java.lang` class to a link (which I think it currently cannot because of missing `package-list`).

Reply:
@jtulach yes, you're right. Previous test only verified the existence of element-list. I have updated the PR to include test cases to verify that element-list is used and links work correctly in generated javadoc.